kinetic/distribution.yaml

```diff
@@ -3513,6 +3513,22 @@ repositories:
       url: https://github.com/ros-teleop/teleop_twist_keyboard.git
       version: master
     status: maintained
+  tesis:
+    doc:
+      type: git
+      url: https://github.com/davidrotor/tesis.git
+      version: branch
+    release:
+      packages:
+      - control
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/davidrotor/tesis.git
```

Comment:
You should use a separate release repository. Currently you have mixed the upstream repo with the release repo.